kubecost-cloud-gcp-marketplace-license.md

```diff
@@ -0,0 +1,36 @@
+# Kubecost Cloud GCP Marketplace Licensing
+
+Kubecost Cloud is [available for licensing on GCP Marketplace](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud) and can be installed in minutes. This guide will take you through licensing through GCP Marketplace, and next steps for setting up Kubecost Cloud. Kubecost currently offers 30 days of Kubecost Cloud free without licensing fees.
+
+{% hint style="info" %}
```

Comment:
This isnt appearing right in the preview

Reply:
What part?